app/views/hello_world/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1 class="alert alert-info this-works">Client Rendering</h1>
+<%= react_component("HelloWorldApp",
+                     @hello_world_props,
+                     generator_function: false,
```

Comment:
Note, generator_function should `true`. This is a bug in the 1.1.1 generator.

By setting `generator_function` to true, we're tell react_on_rails that the HelloWorldApp returns a function which takes props and returns a ReactComponent, rather than exposing a ReactComponent directly. See definition of [HelloWorldApp](https://github.com/shakacode/react_on_rails-generator-results/pull/123/files#diff-1718c1f97315df82ed6fcb45a67f7fe9R10)